worldedit-core/src/main/java/com/sk89q/worldedit/command/argument/OffsetConverter.java

```diff
@@ -62,6 +67,11 @@ public Component describeAcceptableArguments() {
 
     @Override
     public List<String> getSuggestions(String input, InjectedValueAccess context) {
+        if (input.startsWith("^") && context.injectedValue(Key.of(Player.class)).isPresent()) {
```

Comment:
any reason we require `Player` here? would any `Locatable` sender do? (even tho commandblocks' pitch/yaw are probably always `0,0` in theory mobs can be commandsenders too though we don't support them yet)